vendor/priors/gwt.js

```diff
@@ -25,37 +25,28 @@ function fragmentToProps(fragment) {
 function getProm(url) {
```

Comment:
what about the other methods that use Promise?

Reply:
There's nothing wrong with the promise, it's just unnecessary